src/TRestTrackPathMinimizationProcess.cxx

```diff
@@ -223,7 +223,7 @@ void TRestTrackPathMinimizationProcess::BruteForce(TRestVolumeHits* hits, std::v
             if (current_pathweight < min_path) {
                 min_path = current_pathweight;
                 bestPath[0] = s;
-                for (int i = 0; i < vertex.size(); i++) bestPath[i + 1] = vertex[i];
+                for (unsigned int i = 0; i < vertex.size(); i++) bestPath[i + 1] = vertex[i];
```

Comment:
I think that `vector.size()` returns `size_t` instead of `unsigned int`, perhaps would be better to use `auto`

Reply:
I change to `size_t` not clear now for me how to do it with `auto`.